src/interface/app.yaml

```diff
@@ -0,0 +1,302 @@
+openapi: 3.0.1
+info:
+  title: PISP Demo Server Open API
+  description: Based on API Definition.docx Version 1.1 that was confirmed on 2020-05-19.
+    Note - The API supports a maximum size of 65536 bytes (64 Kilobytes) in the HTTP
+    header.
+  license:
+    name: Apache 2.0
+  version: "1.1"
+servers:
+  - url: /
+paths:
+  /health:
+    get:
+      $ref: 'shared.yaml#/paths/~1health/get'
+  /participants:
+    get:
+      tags:
+        - participants
+      summary: 'Lookup the registered DFSPs in Mojaloop'
+      description: The HTTP request GET /participants is used to find the list of registered DFSPs in Mojaloop
+      operationId: getAppParticipants
+      responses:
+        202:
+          $ref: '../../node_modules/@mojaloop/api-snippets/v1.0/openapi3/responses/index.yaml#/202'
```

Comment:
I'd avoid using relative paths with '../..'. Can you at least put this '../../node_modules' into an environment variable like $NODE_MODULES

Reply:
Actually, it is also something that I intended to do... However, I don't think it is currently possible to use internal variable in the Open API file.